docs/environments/simnet.md

```diff
@@ -67,9 +67,9 @@ The following is a command reference to get going:
 
   `$ dcrd --simnet --rpcuser=<username> --rpcpass=<password> --miningaddr=<S....>`
 
-* Instruct dcrd to generate enough initial blocks for the first coinbase to mature:
+* Instruct dcrd to generate enough initial blocks for the first coinbase to mature(genesis block + 16 confirmations):
 
-  `$ dcrctl --simnet --rpcuser=<username> --rpcpass=<password> generate 100`
+  `$ dcrctl --simnet --rpcuser=<username> --rpcpass=<password> generate 17`
```

Comment:
Are we sure 17 is enough blocks for the first coinbase to mature? Was 100 just overkill?

Reply:
Yeah, according to the [devdocs](https://devdocs.decred.org/core-blockchain-concepts/proof-of-stake/stake-enabled-height/) and [dcrd simnet params](https://github.com/decred/dcrd/blob/master/chaincfg/simnetparams.go#L167)
One for generate a block and another 16 to gen confirmations.

Reply:
I think that 100 is overkill and make me confusing, I'm testing here base on (simnet params)[https://github.com/decred/dcrd/blob/master/chaincfg/simnetparams.go#L167], probably we need 18, I will update this PR, give some minutes.